Loadouts/blu_usmc_m16_desert.hpp

```diff
@@ -0,0 +1,259 @@
+    magazines[] = {WEST_GLLAUNCHER_MAG_HE, WEST_GLLAUNCHER_MAG_SMOKE, WEST_PISTOL_MAG };
+    attachments[] = {};
+  };
+  class rhsusf_usmc_marpat_wd_fso : rhsusf_usmc_marpat_wd_officer{// FAC
+    magazines[] = { WEST_GLRIFLE_MAG, WEST_GLRIFLE_MAG_SMOKE, WEST_GLRIFLE_MAG_SMOKE, "rhs_mag_m716_yellow:4", WEST_GLRIFLE_MAG_FLARE, WEST_GLRIFLE_MAG_FLARE, WEST_LEADER_GRENADES };
+    vest[] = {"rhsusf_spc_teamleader"};
+  };
+  class rhsusf_usmc_marpat_wd_autorifleman_m249: rhsusf_usmc_marpat_wd_rifleman {// AR
+    vest[] = {"rhsusf_spc_mg"};
+    weapons[] = {WEST_AR};
+    magazines[] = {WEST_AR_MAG,WEST_PISTOL_MAG,WEST_BASE_GRENADES};
+    handguns[] = {WEST_PISTOL};
+  };
+  class rhsusf_usmc_marpat_wd_autorifleman_m249_ass: rhsusf_usmc_marpat_wd_rifleman {// AAR
+    vest[] = {"rhsusf_spc_light"};
+    backpackItems[] += {WEST_AR_MAG};
+    attachments[] = {WEST_AAR_ATTACHMENTS};
+    linkedItems[] += {BINOS};
+  };
+  class rhsusf_usmc_marpat_wd_riflemanat: Fic_Soldier_Carbine {// RAT
+    magazines[] += {WEST_AT_MAG};
+    launchers[] = {WEST_AT};
+  };
+  class Fic_Spotter: rhsusf_usmc_marpat_wd_rifleman {
+    linkedItems[] += {RANGE_FINDER};
+  };
+  class rhsusf_usmc_marpat_wd_machinegunner: rhsusf_usmc_marpat_wd_autorifleman_m249 {// MMG
+    weapons[] = {WEST_MMG};
+    magazines[] = {WEST_MMG_MAG,WEST_PISTOL_MAG,WEST_BASE_GRENADES};
+    attachments[] = {};
+  };
+  class rhsusf_usmc_marpat_wd_machinegunner_ass: Fic_Spotter {// MMG Spotter/Ammo Bearer
+    vest[] = {"rhsusf_spc_light"};
+    backpackItems[] += {WEST_MMG_MAG2};
+  };
+  class rhsusf_usmc_lar_marpat_wd_machinegunner : Fic_Soldier_Carbine{// HMG
+    backpackItems[] = {};
+    backpack[] = { "RHS_M2_Gun_Bag" };
+  };
+  class rhsusf_usmc_lar_marpat_wd_marksman : Fic_Spotter{// HMG Spotter / Ammo carrier
+    backpackItems[] = {};
+    backpack[] = {"RHS_M2_MiniTripod_Bag"};
+  };
+  class rhsusf_usmc_marpat_wd_smaw: Fic_Soldier_Carbine {// MAT Gunner
+    // Classnames for launcher set at the top of this file
+    vest[] = {"rhsusf_spc_light"};
+    LAUNCHER_GEAR("B_Carryall_mcamo", USMC_MAT_MAG)
+    launchers[] = {USMC_MAT};
+    secondaryAttachments[] = {USMC_MAT_OPTIC};
+  };
+  class rhsusf_usmc_marpat_wd_javelin: Fic_Spotter {// MAT Spotter/Ammo Bearer
+    vest[] = { "rhsusf_spc_light" };
+    LAUNCHER_GEAR("B_Carryall_mcamo", USMC_MAT_MAG2)
+  };
+  class rhsusf_usmc_lar_marpat_wd_crewman : Fic_Soldier_Carbine{ // HAT Gunner
+    LAUNCHER_GEAR("B_Carryall_mcamo", WEST_HAT_MAG)
+    launchers[] = {WEST_HAT};
+  };
+  class rhsusf_usmc_lar_marpat_wd_combatcrewman : Fic_Spotter{// HAT Spotter
+    vest[] = {"rhsusf_spc_light"};
+    LAUNCHER_GEAR("B_Carryall_mcamo", WEST_HAT_MAG)
+  };
+  class rhsusf_usmc_marpat_wd_stinger : Fic_Soldier_Carbine {// SAM Gunner
+    vest[] = { "rhsusf_spc_light" };
+    LAUNCHER_GEAR("B_Carryall_mcamo", WEST_SAM_MAG)
+    launchers[] = {WEST_SAM};
+  };
+  class rhsusf_usmc_marpat_wd_grenadier: Fic_Soldier_Carbine {// Mortar Gunner
```

Comment:
So he can sit on the mortar and driveby people obviously /s

Honestly there's not really a reason, I guess it can be removed.